drivers/ra/fsp/src/bsp/mcu/ra8d2/bsp_linker.c

```diff
@@ -0,0 +1,99 @@
+/*
+* Copyright (c) 2020 - 2025 Renesas Electronics Corporation and/or its affiliates
+*
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#include "bsp_api.h"
+
+/* UNCRUSTIFY-OFF */
+
+/* boot loaded applications cannot set ofs registers (only do so in the boot loader) */
+#if (!defined(CONFIG_BOOTLOADER_MCUBOOT) && !defined(CONFIG_SOC_RA_SECOND_CORE_BUILD))
```

Comment:
As my understanding, the secondary core build check was done by `BSP_CFG_CPU_CORE == 0` below, and `!defined(CONFIG_BOOTLOADER_MCUBOOT)` covered already the non-bootloader image (includes the image for CPU1) check. So, the below checking may be suite enough

```suggestion
#if !defined(CONFIG_BOOTLOADER_MCUBOOT)
```